velox/connectors/hive/iceberg/IcebergSplit.cpp

```diff
@@ -68,6 +71,7 @@ HiveIcebergSplit::HiveIcebergSplit(
           _fileFormat,
           _start,
           _length,
+          _properties,
```

Comment:
`{}` should work here? other places as well.

Reply:
This doesn't compile. Same for the other comments.